configs/components/rubygem-puppet-resource_api.rb

```diff
@@ -1,5 +1,5 @@
 component "rubygem-puppet-resource_api" do |pkg, settings, platform|
   pkg.version "1.8.0"
-  pkg.md5sum "f5f8d8414c27f6f015b70f12a0c27fe83878bbac346746d82c27858acf7378f5"
```

Comment:
I tried making this a suggestion but it ended up making a new commit. I think you copied over a sha256 when it should be a md5 https://artifactory.delivery.puppetlabs.net/artifactory/webapp/#/artifacts/browse/tree/General/rubygems__remote-cache/gems/puppet-resource_api-1.8.0.gem